main/zoe/guide/contracts/atomic-swap.md

```diff
@@ -50,20 +50,23 @@ const aliceProposal = harden({
 const alicePayments = { Asset: aliceMoolaPayment }
 ```
 
-In order for Alice to escrow with Zoe she needs to redeem her invite. Once Alice redeems her invite she will receive a `seat` and a promise that resolves to her payout.
+In order for Alice to escrow with Zoe she needs to redeem her invite.
```

Comment:
This still has "redeem".  I have some thoughts on rewrite, but I'll just make the comments here.

Reply:
I thought we were ok with the term redeem, just not the call zoe.redeem since that doesn't exist anymore. I don't mind changing it to use